docs/mkdocs.yml

```diff
@@ -65,13 +65,15 @@ nav:
     - Configuration: start/config.md
     - Docker:
         - Introduction: start/docker.md
-        - Development: start/docker_dev.md
```

Comment:
I think the setup instructions should stay here and be linked in an overview dev doc.

Reply:
And where should I put the devcontainer doc? Directly at the same level of docker?

Reply:
I think it is good where it is. We are dragging around the idea of some better dev docs for a while now. This PR can be the start and I will try to formulate something for a general getting started from this: https://github.com/inventree/InvenTree/issues/4793#issuecomment-1540758748

Reply:
Sorry, don’t get how I should do it, maybe you can push it like you want?

Reply:
I can if that is ok with you; I do not like pushing to other ppl. branches, feels like overstepping

Reply:
Yeah, absolutely fine.